src/main/java/Lottery.java

```diff
@@ -0,0 +1,11 @@
+import java.util.Random;
+public class Lottery {

```

Comment:
there should be space between the import statement and the start of a class definition, please make sure you use IDE autoformat feature before committing in each file